endpoints/api/mirror.py

```diff
@@ -231,7 +232,7 @@ def get(self, namespace_name, repository_name):
       'sync_retries_remaining': mirror.sync_retries_remaining,
       'sync_status': mirror.sync_status.name,
       'root_rule': {
-        'rule_type': 'TAG_GLOB_CSV',
+        'rule_type': 1,
```

Comment:
I believe that we should continue using `TAG_GLOB_CSV` or some human-friendly String.
The integer `1` is simply an internal enumeration and probably shouldn't be exposed through the API.

Reply:
@kurtismullins other tables seem to use a "kind" that maps a string to an int. should we future proof the rules and change "rule_type" to "rule_kind" in the API? we wouldn't change the db columns but could if we wanted in the future

https://docs.quay.io/api/swagger/#!/repository/listRepos
repo_kind

Reply:
Replying to myself... yes, if we're going to use strings let's stick with the `kind` pattern.

Reply:
@kurtis updated